.gitignore

```diff
@@ -10,6 +10,8 @@
 
 # Test binary, built with `go test -c`
 *.test
```

Comment:
All binaries are built in `/bin`

Reply:
that was default .gitignore for `go` application, but oki

Reply:
We have to be organized and keep it tidy. Creating binaries everywhere has side effects, e.g the cache determination in the earthly when copying directories with `COPY --dir` and expecting only go files.

What is the use case for running `go test -c`? Why would you use it?

Reply:
As I said, this is template from https://github.com/github/gitignore
[go .gitignore](https://github.com/github/gitignore/blob/main/Go.gitignore) 
There is also [vscode extension](https://marketplace.visualstudio.com/items?itemName=codezombiech.gitignore) that I used : 

I simply did not argue with it contents and presumed that everything is good there